letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
I think that `letsencrypt --standalone` is a pretty simple command that should just do what it seems like it should do.  I don't think there's a conceptual problem with the internal semantics of a CLI tool being slightly different when it receives a flag, if that's what makes logical sense.

That's especially true when we're talking about the *default* selection of subcommands, because the user will often not have much of a model of what the subcommands exactly are and what they exactly do.  And if they do have that model, the help is [very clear](https://github.com/letsencrypt/letsencrypt/pull/1038/files#diff-54b2f468d57ef7436008bd031bc89711R74).